### PR TITLE
Add dudes as pop up

### PR DESCRIPTION
**Pull request by EliranGooner**

### CODE REVIEW

File: app/views/user_units/new.html.erb [hunk 1]
```diff
@@ -79,7 +110,7 @@
                         id: 'add-ten-units' do %>
                         <div class='gem'>
                           90<br>
-                          +10 <img src="https://pbs.twimg.com/profile_images/531994916181012480/EBap51cO_400x400.png" alt="" style="width:40px;height: 40px">
+                          +10 <img src="castle4_ioe9t4" alt="" style="width:40px;height: 40px">
```
Comment (bluepostit) [inline app/views/user_units/new.html.erb]: Doesn't seem right... You probably want a `cl_image_tag`

File: app/assets/stylesheets/pages/_card.scss [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+  }
+  &:hover .infos {
+    transform:translateY(-$thumb-height);
+    .seats, .txt, .details {
+      opacity:1;
+    }
+  }
+}
+@import url('https://fonts.googleapis.com/css?family=Advent+Pro&display=swap');
+.category {
```
Comment (bluepostit) [inline app/assets/stylesheets/pages/_card.scss]: Too generic! I suggest you use a different class name - something more specific.

File: app/assets/stylesheets/pages/_card.scss [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+}
+.note {
+  margin-top: 30px;
+  color:$white;
+  font-size: 1rem;
+  font-family: $merriweather;
+  line-height: 1.5;
+  text-align: center;
+}
+.card {
```
Comment (bluepostit) [inline app/assets/stylesheets/pages/_card.scss]: Too generic! I suggest you rename it to something more specific.

File: app/assets/stylesheets/pages/_card.scss [hunk 4]
```diff
@@ -0,0 +1,171 @@
+.cardbox {
```
Comment (bluepostit) [inline app/assets/stylesheets/pages/_card.scss]: Too generic! I suggest you rename to something more specific.

**Comment by bluepostit** [comment]
@christonpeterson @EliranGooner I've suggested a few CSS class name changes. IMPORTANT to note that when you change those, you will also need to change references to them in the HTML...

File: app/assets/images/iron1.pixil [hunk 5]
```diff
@@ -0,0 +1 @@
...cm5y7IqFNHFsDxg+7JkDFudq2QIhm5iwI6XZDBZw6qA/0iQRUM4HhHJWkUZMg4u3sOyZ8zgAKTdVG0IefqtrZZZbopts3JYHvzccNoyJCxd6OQJjIIkM2pDmubpzwPGTZmiEpGRYZg0E4hQgQ+FJgK0rOAJRlc4bPq834TGYgXUIrZ2sJljY4MmWs7hcgb0ikyGCssMAQU+aJk0C0xZqAvkMBkyYCrGisZMp9WCjnEnz1tIuC53db56IP39bL0dxBbj2WAzX5pK+UI3uhzzGTIvDbxkcOiW96r33/7zeWXXnxBPzLgxwZffv5Z9fjGGbXl5dcvvVXdvPtf9fedW3qcOftMCdsp29uX3tBr+2qEFfgBAwhDv2MnQ+Z1BD+LtCgp+Op80sOMM1yMjQO/X/9JW9QphXVRCfEGSrFuCna6MrhKcZ+jjBkyjamHyS4moYG9jhi76wGIOdAgpkQK6jHwrwsZIKetQqisIikM7nwm4Iq2agFJiCuWFJJmCUG9PKF/xCv5WsT1C0MZN3GamXehEP5IkRS6GZIBUGlDmeCTFF7bHRXq5WkVyZA5dvo3dVUbVUEALRGwWTJwDRVQCSSD17YuyrxGzFg1ZQgUGku6/ps6SOEHzPa/mIB9AhTlnKi967BOEu4RfOa4AzvIgHtbJTclU0uBvY+/h4AU+HRsQ5El8AEoD9jtqsc1n1FIAuoysYyHUel3ZWKGzC+RIeXNrhViOwdohyDlihQQ2AmqXGrKr2m3ua1z4g5XlgyZd6cxxJLBQA/wNLZQLbkqAD5scENfffcDLlOiUmBYBzJkmsBtclpCjl2YHqIW/bdmeXbRf2fGNV5USqBO17x38gIT7XSwq573EUNKSgGYp2ILFWDdktRLro0vK8W00m5K5kePslSF2BWelGKVQIXARsXABpIk2farfD3oZBMxJGDNyRhMIXaFKykgxB5rqAx6gEEVwkFMkbLGZLhQyBQp606GykNOBCXyCSKD4jDoj8diOCUGXRJBigNlcHEGGY7ImAwlYoirRelqMJTtGucqkiBl4itc4OBiEGusiBz/UIi3xZAzFNfDuq9QSChk2BXo3QOEQkIhoRAgUKdURafuZtgHWDwB+gCgh0J8gd4ogsabDUxGu35I1l4D3H7AXQjXhRqFcmp3SW3xDIV4Wlz/A/4ERejih27aAAAAAElFTkSuQmCC"}
```
Comment (christonpeterson) [inline app/assets/images/iron1.pixil]: To do: delete this line

**Pull request merged**